src/components/SidebarFeeds.js

```diff
@@ -1,35 +1,56 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
make sure to remove unused imports (do you have a linter set up to flag these in your IDE?)

Reply:
done. I do, but it's not as intrusive with these errors a it could be. I'll check on configuring it better